dbuf/dbuf/retry_forever.py

```diff
@@ -0,0 +1,19 @@
+import asyncio
+
+from .logging import log
+
+
+async def retry_forever(f, msg=None):
+    i = 0
+    while True:
+        try:
+            await f()
+            break
+        except Exception as exc:
+            if msg:
+                log.warning(msg(exc))
+        i = 1.1 ** i
+        await asyncio.sleep(0.100 * i)
+        # max 44.5s
+        if i < 64:
```

Comment:
So I think these are bad defaults for this method.  They came from the batch client that was trying to minimize the delay introduced by polling I think.  I'd prefer increasing the delay by 2 and adding in some randomness (I've been using random.uninform(0.7, 1.3)) elsewhere.

Reply:
OK, I'm using sleep_and_backoff